coordinator_test.go

```diff
@@ -34,6 +35,7 @@ func TestNewCoordinator(t *testing.T) {
 				{tableName: "A", parentTableName: ""},
 				{tableName: "B", parentTableName: ""},
 			},
+			indexes: nil,
```

Comment:
Zero value for the slice is `nil`, so we can omit that.

Reply:
applied!